composer/core/time.py

```diff
@@ -212,6 +215,20 @@ def from_duration(cls, duration: float) -> Time:
         """
         return cls(duration, TimeUnit.DURATION)
 
+    @classmethod
+    def from_second(cls, second: int) -> Time:
+        """Create a :class:`Time` with units of :attr:`TimeUnit.SECOND`.
```

Comment:
Why is there no from_hour, from_minute?

Reply:
hmm, i kind of think that as a user I should be able to specify my time in some format string style, e.g. `1h5m30s`, with something like https://stackoverflow.com/questions/4628122/how-to-construct-a-timedelta-object-from-a-simple-string.

Can we just have one `from_timedelta` or something?

Reply:
Imo it's fine to just specify 1 time unit, but im fine with Daniel's as well. most users likely want to reason w like ckpt every 1 hr not every 3600s